app/models/fee_paid.rb

```diff
@@ -0,0 +1,5 @@
+class FeePaid < Base
+  attribute :answer, Boolean
+
+  validates :answer, inclusion: { in: [true, false] }
```

Comment:
This should have a date fee paid too

[link](http://ministryofjustice.github.io/fr-prototypes/public/journey-7/refund.html)